addons/okr-daily-script.kp/knowledge.md

```diff
@@ -0,0 +1,229 @@
+---
+title: addon_aggregates derived dataset script
+authors:
+- Ben Miroglio
+tags:
+- add-ons
+- okr
+- derived dataset
+created_at: 2017-02-08 00:00:00
+updated_at: 2017-02-27 13:30:50.584980
+tldr: script to be run daily that contructs the addon_aggregates table in re:dash
+---
+# Add-ons 2017 OKR Data Collection
+
+Some OKRs for 2017 can be feasibly collected via the `addons` and `main_summary` tables. These tables are huge and aren't appropriate to query directly via re:dash. This script condenses these tables so that the result contains the least data possible to track the following OKRs:
+
+* **OKR 1: Increase number of users who self-install an Add-on by 5%**
+* **OKR 2: Increase average number of add-ons per profile by 3%**
+* **OKR 3: Increase number of new Firefox users who install an add-on in first 14 days by 25%**
+
+These OKRs, in addition to other add-on metrics, are tracked via the [Add-on OKRs Dashboard](https://sql.telemetry.mozilla.org/dashboard/add-on-okrs_1#edit_dashboard_dialog) in re:dash.
+
+
+```python
+from __future__ import division
+import pyspark.sql.functions as fun
+import pyspark.sql.types as st
+import pandas as pd
```

Comment:
Please remove unused imports: `pd`, `plt`, `seaborn` and the setup calls on line 35 and 36.